talkative-tangs/bangazon-react
# OE-Managers

## Link to Ticket
[Issue #13](https://github.com/talkative-tangs/Bangazon-API/issues/13)

## Description of Proposed Changes

-Reorganized our file structure.
-Added Semantic-UI
-Added React Router
-Added API Managers
-Added "Home" Page
-Added Customers List
-Setup Promise and Initial State
-More that I can't think of.

## Steps to Test

-Make sure to NPM Install from Banagazon-React top directory.
-Begin at localhost:3000/home with your django server set to 8000.
-Click on Ecommerce.
-Click on Customers List.
-Ensure list of customers is available.

```sh
git fetch --all
git checkout oe-managers

```

## Impacted Areas in Application

List general components of the application that this PR will affect:

* Components
* Managers
* App.Js
* Index.Js
* More

## Mentions @username

@robbyhecht  
@BryanNilsen
@laboyd001
